pylint/test/unittest_checker_spelling.py

```diff
@@ -35,12 +35,21 @@ class TestSpellingChecker(CheckerTestCase):
                         "spelling dictionaries")
     @set_config(spelling_dict=spell_dict)
     def test_check_bad_coment(self):
-        with self.assertAddsMessages(
-            Message('wrong-spelling-in-comment', line=1,
-                    args=('coment', '# bad coment',
-                          '      ^^^^^^',
-                          "comet' or 'comment' or 'moment' or 'foment"))):
-            self.checker.process_tokens(tokenize_str("# bad coment"))
+        try:
```

Comment:
If check is fragile and message changes based on local dict available on machine, assertion should ignore "suggestion" section entirely. Or maybe you can ask enchant library for "expected suggestions" and use it for building "expected" messgae.